zipkin-cassandra-core/build.gradle

```diff
@@ -4,6 +4,10 @@ repositories {
     mavenCentral()
 }
 
+configurations {
+    all*.exclude group: 'org.scala-lang'
```

Comment:
needed because the top-level build has this in allProjects, and we can't remove that without breaking the idea goal...